### PR TITLE
Bots created with frankenstein

### LINKED ISSUE #79: Would like to put my example bot in README  ?
Hi @ayrat555 , Awesome library.

I was wondering if you would like us to put my bot in the README.
My bot is done with this library and i think is a good example to beginners.

https://github.com/pxp9/weather_bot_rust

Maybe you can put it in this README section.
image

Thank you for this awesome tool for building creative and wonderful bots.

### PR DESCRIPTION
**Pull request by pxp9**
This closes #79

### CODE REVIEW

**Review by EdJoPaTo — CHANGES_REQUESTED** [review]
Can you sort them alphabetically? Maybe put the El Monitorro at first and sort the rest so the creator of this crate is also first, but the rest has no preferred order and is simply just sorted?

**Comment by pxp9** [comment]
> Can you sort them alphabetically? Maybe put the El Monitorro at first and sort the rest so the creator of this crate is also first, but the rest has no preferred order and is simply just sorted?

@EdJoPaTo 
Done.

File: README.md [hunk 1]
```diff
@@ -197,7 +197,13 @@ AsyncApi::builder().api_url(api_url).client(client).build()
...
 Frankenstein implements all telegram bot api methods. To see which parameters you should pass, check [docs.rs](https://docs.rs/frankenstein/0.19.1/frankenstein/api_traits/telegram_api/trait.TelegramApi.html#provided-methods)
 
-You can check out a real world bot created using this library - [El Monitorro](https://github.com/ayrat555/el_monitorro). El Monitorro is a feed reader bot.
+You can check out real-world bots created using this library:
+
+- [El Monitorro](https://github.com/ayrat555/el_monitorro) - RSS/Atom/JSON feed readeer.
+- [ledit](https://github.com/jflessau/ledit) - task assignment bot.
+- [subvt-telegram-bot](https://github.com/helikon-labs/subvt-backend/tree/main/subvt-telegram-bot) - A telegram bot for the validators of the [Polkadot](https://polkadot.network/) and [Kusama](https://kusama.network/).
+ - [wdr-maus-downloader](https://github.com/EdJoPaTo/wdr-maus-downloader) - checks for a new episode of the WDR Maus and downloads it.
+ - [weather_bot_rust](https://github.com/pxp9/weather_bot_rust) - A telegram bot that provides weather info around the world.
```
Comment (EdJoPaTo) [inline README.md:206]: Looks like here are spaces in the front
```suggestion
- [wdr-maus-downloader](https://github.com/EdJoPaTo/wdr-maus-downloader) - checks for a new episode of the WDR Maus and downloads it.
- [weather_bot_rust](https://github.com/pxp9/weather_bot_rust) - A telegram bot that provides weather info around the world.
```

**Review by EdJoPaTo — CHANGES_REQUESTED** [review]

**Review by ayrat555 — APPROVED** [review]
@pxp9 @EdJoPaTo thank you!

**Pull request merged**